app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt

```diff
@@ -0,0 +1,119 @@
+package io.homeassistant.companion.android.settings.wear.views
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mikepenz.iconics.typeface.library.community.material.CommunityMaterial
+import io.homeassistant.companion.android.common.data.prefs.impl.entities.TemplateTileConfig
+import io.homeassistant.companion.android.settings.views.SettingsRow
+import io.homeassistant.companion.android.common.R as commonR
+
+@Composable
+fun SettingsWearTemplateTileList(
+    templateTiles: Map<Int, TemplateTileConfig>,
+    onTemplateTileClicked: (tileId: Int) -> Unit,
+    onBackClicked: () -> Unit
+) {
+    Scaffold(
+        topBar = {
+            SettingsWearTopAppBar(
+                title = { Text(stringResource(commonR.string.template_tiles)) },
+                onBackClicked = onBackClicked,
+                docsLink = WEAR_DOCS_LINK
+            )
+        }
+    ) { padding ->
+        Column(
+            Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(padding)
+                .padding(all = 16.dp)
```

Comment:
The title / rows already have padding so you don't actually need this most of the time. 

You do need some padding for the 'no tiles yet' message (line 46), so I suggest setting the 16dp padding on that text instead.

Reply:
Like this?

```diff
diff --git app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
index f77983e8..85329f96 100644
--- app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
+++ app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTileList.kt
@@ -40,10 +40,13 @@ fun SettingsWearTemplateTileList(
             Modifier
                 .verticalScroll(rememberScrollState())
                 .padding(padding)
-                .padding(all = 16.dp)
         ) {
             if (templateTiles.entries.isEmpty()) {
-                Text(stringResource(commonR.string.template_tile_no_tiles_yet))
+                Text(
+                    text = stringResource(commonR.string.template_tile_no_tiles_yet),
+                    modifier = Modifier
+                        .padding(all = 16.dp)
+                )
             } else {
                 Row(
                     modifier = Modifier
```